keyboards/nullbitsco/scramble/scramble_led.c

```diff
@@ -0,0 +1,126 @@
+/* Copyright 2021 Jay Greco
```

Comment:
This file shouldnt really exist, and the logic should be split between `keyboards/nullbitsco/scramble/v1/v1.c` and `keyboards/nullbitsco/scramble/v2/v2.c`

Reply:
Good feedback -- fixed!